frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts

```diff
@@ -96,7 +96,7 @@ describe('Project Dashboard', () => {
       expect(items.get(1).getText()).toEqual('Requester');
       expect(values.get(1).getText()).toEqual('kube:admin');
       expect(items.get(2).getText()).toEqual('Labels');
-      expect(values.get(2).getText()).toEqual('No labels');
+      expect(values.get(2).getText()).toEqual(`kubernetes.io/metadata.name = ${testName}`);
```

Comment:
I don't think there is a space before and after the `=`?

Reply:
Output from the [failed CI test](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_console/8497/pull-ci-openshift-console-master-e2e-gcp-console/1379815109647405056/artifacts/e2e-gcp-console/test/artifacts/gui_test_screenshots/protractor-report.html) shows `Expected 'kubernetes.io/metadata.name = test-glrcm' to equal 'No labels'`